### ISSUE TITLE
Nil pointer error with snipkit 1.3.1 and 1.3.0, not 1.2.0

### ISSUE THREAD

**Issue by enlewof** [lemoony/snipkit#128]
When I run snipkit 1.3.1 on my intel mac (running Mac os 12.6), I seem to be getting this error:

```snipkit
runtime error: invalid memory address or nil pointer dereference
```

When I run 1.2.0 though, I don't see any errors.   Any suggestions on how I can try to debug this further?  Thanks.

**Comment by lemoony** [comment]
Sorry for the delayed reply.

First thing would be to have look at the logs. For that, it is best to set the log level explicitily: `snipkit -l trace`
You can find the logs in `$SNIPKIT_HOME/.log`

If you haven't set `SNIPKIT_HOME` you can detekt the path currently used via `snipkit info`.

My guess would be that it has something to do with the config file. Do you mind sharing that as well?

**Comment by enlewof** [comment]
Sorry for the delay as well.  Here are the logs:
```16:25:04 DBG 1 service.go:72 > SnipKit Home: /Users/alew/.snipkit
16:25:04 TRC 1 themes.go:63 > Chosen theme: default.dark
16:25:04 DBG 1 service.go:72 > SnipKit Home: /Users/alew/.snipkit
16:25:04 TRC 1 themes.go:63 > Chosen theme: default.dark
16:25:04 TRC 1 themes.go:63 > Chosen theme: default.dark
16:25:04 TRC 1 tui.go:107 > Color profile: 5
16:25:04 ERR 1 root.go:157 > error="runtime error: invalid memory address or nil pointer dereference" Exited with panic error: runtime error: invalid memory address or nil pointer dereference
goroutine 1 [running]:
github.com/phuslu/log.stacks(0x0)
        /home/runner/go/pkg/mod/github.com/phuslu/log@v1.0.80/logger.go:2091 +0x8a
github.com/phuslu/log.(*Entry).Stack(0xc0003506c0)
        /home/runner/go/pkg/mod/github.com/phuslu/log@v1.0.80/logger.go:1514 +0xac
github.com/lemoony/snipkit/cmd.handlePanic()
        /home/runner/work/snipkit/snipkit/cmd/root.go:157 +0xb9
panic({0x184faa0, 0x1fa0dd0})
        /opt/hostedtoolcache/go/1.18.3/x64/src/runtime/panic.go:838 +0x207
github.com/lemoony/snipkit/internal/managers.createMassCodeConfig({{0x1af0ae0, 0x1ff2640}, 0x0, 0x0, 0x0, 0x0, 0x0}, {0x0, 0x0, 0xc00029b300, ...})
        /home/runner/work/snipkit/snipkit/internal/managers/provider.go:142 +0x12d
github.com/lemoony/snipkit/internal/managers.providerImpl.CreateManager({{0x1aef0c0?, 0xc0001343b8?}}, {{0x1af0ae0, 0x1ff2640}, 0x0, 0x0, 0x0, 0x0, 0x0}, {0x0, ...})
        /home/runner/work/snipkit/snipkit/internal/managers/provider.go:43 +0x427
github.com/lemoony/snipkit/internal/app.NewApp({0xc000343b88, 0x3, 0x9?})
        /home/runner/work/snipkit/snipkit/internal/app/app.go:92 +0x3f4
github.com/lemoony/snipkit/cmd.getAppFromContext({0x1aee980, 0xc00013a010})
        /home/runner/work/snipkit/snipkit/cmd/root.go:59 +0x265
github.com/lemoony/snipkit/cmd.glob..func5(0x1fab800?, {0xc0002c6b48?, 0x4?, 0x0?})
        /home/runner/work/snipkit/snipkit/cmd/exec.go:12 +0x2a
github.com/lemoony/snipkit/cmd.glob..func10(0x1fab800, {0xc0002eb620, 0x0, 0x2})
        /home/runner/work/snipkit/snipkit/cmd/root.go:96 +0x151
github.com/spf13/cobra.(*Command).execute(0x1fab800, {0xc000118190, 0x2, 0x2})
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:856 +0x67c
github.com/spf13/cobra.(*Command).ExecuteC(0x1fab800)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:974 +0x3b4
github.com/spf13/cobra.(*Command).Execute(...)
        /home/runner/go/pkg/mod/github.com/spf13/cobra@v1.4.0/command.go:902
github.com/lemoony/snipkit/cmd.Execute()
        /home/runner/work/snipkit/snipkit/cmd/root.go:118 +0x45
main.main()
        /home/runner/work/snipkit/snipkit/main.go:26 +0x53
```

And my config:
```
❯ cat config.yaml
version: 1.1.0
config:
  style:
    # The theme defines the terminal colors used by Snipkit.
    # Available themes:default(.light|.dark),simple.
    theme: default
  # Your preferred editor to open the config file when typing 'snipkit config edit'.
  editor: "" # Defaults to a reasonable value for your operation system when empty.
  # The command which should run if you don't provide any subcommand.
  defaultRootCommand: "exec" # If not set, the help text will be shown.
  scripts:
    # The path to the shell to execute scripts with. If not set or empty, $SHELL will be used instead. Fallback is '/bin/bash'.
    shell: ""
    # Defines how parameters are handled. Allowed values: SET (sets the parameter value as shell variable) and REPLACE (replaces all occurrences of the variable with the actual value)
    parameterMode: ""
    # If set to true, any comments in your scripts will be removed upon executing or printing.
    removeComments: false
  manager:
    pet:
      # Set to true if you want to use pet.
      enabled: true
      # List of pet snippet files.
      libraryPaths:
        - /Users/alew/.config/pet/snippet.toml
      # If this list is not empty, only those snippets that match the listed tags will be provided to you.
      includeTags: []
```

I will point out that my pet snippet.toml is a symlink to a git repo that contains my dotfiles and that is also the case with my snipkit config.

```
total 0
lrwxr-xr-x  1 alew  staff    56B Oct 17 08:38 config.yaml -> /Users/alew/Documents/code/dotfiles/.snipkit/config.yaml

❯ pwd
/Users/alew/.snipkit
```

**Comment by lemoony** [comment]
Hi @enlewof , I've released version 1.3.2 which should fix the issue. Feel free to report if there are any other issues.

**Comment by enlewof** [comment]
Thanks @lemoony.  Looks like it is working once again.
